### ISSUE TITLE
deviceId instead of alias?

### ISSUE THREAD

**Issue by jacobkossman** [adumont/tplink-cloud-api#34]
Is it possible to use the device id (which is unique and not messy with spaces etc) instead of the alias?

eg: `tplink.getHS100(DEVICEID).toggle();`

**Comment by jacobkossman** [comment]
```
app.post('/api/v1/device/toggle', async (request, response) => {
  var myTPLink = await tplink.login(TPLINK_USER, TPLINK_PASS,TPLINK_TERM);
  let deviceList = await myTPLink.getDeviceList();
  let obj = deviceList.find(o => o.deviceId === request.body.deviceID);
  await myTPLink.getHS100(obj.alias).toggle();
  response.sendStatus(200);
});
```

I've resorted to doing this so I can create an API without having to use the device alias.

`let obj = deviceList.find(o => o.deviceId === request.body.deviceID);` is the magic line.

**Comment by adumont** [comment]
Thanks, good idea. I've added it in f9cd26d, published as `tplink-cloud-api@0.6.0`.
